Tests/Functional/Asynchronous/Use/main.swift

```diff
@@ -56,7 +60,11 @@ class AsyncAwaitTests: XCTestCase {
     
     override func setUp() async throws {}
     
-    override func tearDown() async throws {}
+    override func tearDown() async throws {
```

Comment:
Makes sense, although I wonder if it would be ever-so-slightly more robust to place this flush in an override of the class-level `tearDown` method rather than this instance one? The class one isn't `async` but it runs later, after all tests and their `tearDown` methods complete, so it would theoretically be more inclusive in case there were stdout generated after an instance `tearDown`. I.e.
```suggestion
    override class func tearDown() {
```

Reply:
I think that I am going to disable the test on Windows instead.  I noticed that the runtime was triggering asserts, and I think that is the core issue and that the flush might be a red herring.  Sorry about the change of approach.